src/freenas/usr/local/lib/middlewared_truenas/plugins/failover.py

```diff
@@ -1425,8 +1448,11 @@ def journal_sync(middleware):
             journal_sync = JournalSync(middleware, sql_queue, journal)
             while True:
                 journal_sync.process()
-        except Exception:
-            logger.warning('Failed to sync journal', exc_info=True)
+        except Exception as e:
```

Comment:
except OSVersionMismatch as e:
    logger...
except Exception:
    logger

Reply:
Also we will most definitely keep getting that message on log every 5 seconds, which seems less than ideal

Reply:
Yeah we will get logs every 5 seconds so I can either change the log message to:
1. logger.debug
or
2. don't raise a custom exception and do what we do when `failover.status != 'MASTER'` and clear the journal?

Reply:
or 3. use a variable to log it consecutively only once


1. wont solve it, only .trace would but then we will miss it entirely